apps/console/src/features/identity-providers/components/wizards/steps/identity-create-steps/general-settings.tsx

```diff
@@ -47,16 +52,98 @@ export const GeneralSettings: FunctionComponent<GeneralSettingsWizardFormPropsIn
         [ "data-testid" ]: testId
     } = props;
 
+    const [ isNameValid, setIsNameValid ] = useState(false);
```

Comment:
Shall we add the state type as well here?

Reply:
Resolved by 90dd7d8d